doc/user/compatibility.md

```diff
@@ -35,15 +35,7 @@ In the C API, the preprocessor macro `TRUFFLERUBY` is defined, which can be chec
 
 ## Ruby 3.x Features
 
-Most features of Ruby 3.2 and earlier are supported in TruffleRuby.
-However some features are not yet implemented.
-See the following issues for details:
-
-* [Ruby 3.2 features](https://github.com/oracle/truffleruby/issues/3039)
-* [Ruby 3.1 features](https://github.com/oracle/truffleruby/issues/2733)
-* [Ruby 3.0 features](https://github.com/oracle/truffleruby/issues/2453)
-* [Pattern Matching](https://github.com/oracle/truffleruby/issues/3332)
-* [Endless methods](https://github.com/oracle/truffleruby/issues/3038)
```

Comment:
These 2 lines should be removed because the issues are fixed, the rest should stay there.

Reply:
I’ve reviewed the trackers. The 3.0 and 3.1 trackers were closed as we planned to “address the remaining items when it becomes possible or important for compatibility”, but I missed that the 3.2 one is still half-open as it’s closed with “should still be tracked here”.
It looks like the 3.0 and 3.1 ones are no longer updated (I see that the pattern matching and endless methods boxes are still unchecked), but the 3.2 tracker had updates last month.

Perhaps it’s safe to also remove the 3.0 and 3.1 tracker bullet points? Is the 3.2 tracker up-to-date?

With so few items remaining open in all three trackers, it might be more convenient simply include them in the (future) 3.3 tracker.

Reply:
> It looks like the 3.0 and 3.1 ones are no longer updated (I see that the pattern matching and endless methods boxes are still unchecked), but the 3.2 tracker had updates last month.

Actually we should update the 3.0 and 3.1 issues, I'll do that, thanks for checking.
The idea is to keep all these 3 issues because they are not 100% done. Sometimes people come and ask how come you say compatibility with X.Y but feature F is not available?
The answer can be found partially in those issues and because we didn't expect F to be used much or there are issues to implement it, etc.

> With so few items remaining open in all three trackers, it might be more convenient simply include them in the (future) 3.3 tracker.

We don't mix issues from one version with another, that would be messy and these issues are based on the corresponding NEWS file from CRuby.